x-pack/plugins/triggers_actions_ui/public/index.ts

```diff
@@ -11,7 +11,13 @@ export { AlertsContextProvider } from './application/context/alerts_context';
 export { ActionsConnectorsContextProvider } from './application/context/actions_connectors_context';
 export { AlertAdd } from './application/sections/alert_form';
 export { ActionForm } from './application/sections/action_connector_form';
-export { AlertAction, Alert, AlertTypeModel, ActionType } from './types';
+export {
+  AlertAction,
+  Alert,
+  AlertTypeModel,
+  AlertTypeParamsExpressionProps,
```

Comment:
`AlertTypeParamsExpressionProps` doesn't seem to be referenced in this module.  I assume you would have gotten a lint error on this

Reply:
It's an export not an import ;)